src/index.js

```diff
@@ -95,8 +94,13 @@ export default (opt = {}) => {
 					writeFileSync(file, code);
 				}
 
+				
 				let src = isURL(file) ? file : absolutePathPrefix + relative(destDir, file).replace(/\\/g, '/');
-
+				if (prefix) { 
+					const filename = file.split('/').slice(-1)[0];
+					const divide = filename.slice(-1) === '/' ? '' : '/';
+					src = prefix + divide + filename;
```

Comment:
don't really get why the divider is added based on last character of filename.
on the other hand, maybe url.resolve could help to generage the url?

Reply:
Yeah my bad, that should be `prefix.slice(-1)`. But I don't think `url.resolve` can help because this is mainly for someone forgetting to add a slash at the end of the prefix, while `url.resolve` would repalce the last path on that case.

Reply:
BTW do you think `publicPath` would be a better option name?

Reply:
hmm, looks good, let's take this name.